### DISCUSSION TITLE
CP doens't work after deploy

### DISCUSSION THREAD

**Discussion by AnnelynnTruyens** [statamic/cms#9487] [Q&A] [answered]
I just deployed my statamic site using Digital Ocean App Platform.

My frontend site works, but I can't access my control panel anymore. I get a 500 server error when trying to access it.
I got this error before with my frontend as well, because my env variables were not yet added. So I assume it's a problem with my env variables, does anyone know which one could be the problem for the control panel?

**Comment by duncanmcclean** [comment]
Are you able to access the Laravel log? It'll usually tell you what error is occurring, then you can fix it from there.

Normally, you'd find your Laravel log file in `storage/logs` but I'm not sure where/if it's accessible within DigitalOcean App Platform.

**Reply by AnnelynnTruyens** [reply]
I can see runtime logs which gives me this information:
<img width="776" alt="Screenshot 2024-02-09 at 11 45 50" src="https://github.com/statamic/cms/assets/115149397/45cded25-e9b6-440c-ad93-d9eca7cd1961">

**Reply by duncanmcclean** [reply]
Unfortunately, that's the wrong sort of log. It just says which URLs were hit and by *what* (eg. browser). It's not the Laravel log file, which is needed to figure out the error.

**Reply by AnnelynnTruyens** [reply]
Then I have no idea where or if I can find them on DigitalOcean. I can only see the local log files in my local folder, but not the ones of the deployed website.

**Reply by duncanmcclean** [reply]
I've never used DigitalOcean App Platform before but from the outside it doesn't *look* like it's possible to see "application" logs. 🤔 

Another option would be to temporarily enable debug mode but I would make sure you disable it as soon as possible since it's a big security risk.

Alternatively, you could try using something like [Forge](https://statamic.dev/deploying/laravel-forge) or [Ploi](https://statamic.dev/deploying/ploi) with a DigitalOcean droplet. They give you much more control over your server.

**Reply by AnnelynnTruyens** [reply, marked as answer]
I temporarily enabled debug mode and found the problem. My cp.css and cp.js were not found with the build as I'm not using those files, but they were put in the boot function in the AppServiceProvider.php. I deleted that part of the code in that file and the problem is now solved. Thank you for your help!
